operator/src/main/java/oracle/kubernetes/operator/helpers/EventHelper.java

```diff
@@ -157,6 +159,20 @@ public NextAction onSuccess(Packet packet, CallResponse<V1Event> callResponse) {
             .ifPresent(dpi -> dpi.setLastEventItem(eventData.eventItem));
         return doNext(packet);
       }
+
+      @Override
+      public NextAction onFailure(Packet packet, CallResponse<V1Event> callResponse) {
+        if (isNotAuthorizedForNamespaceWatchingStoppedEvent(callResponse)) {
+          LOGGER.info(MessageKeys.CREATING_EVENT_UNAUTHORIZED,
+                  eventData.eventItem.getReason(), eventData.getResourceName());
+        }
+        return super.onFailure(packet, callResponse);
+      }
+
+      private boolean isNotAuthorizedForNamespaceWatchingStoppedEvent(CallResponse<V1Event> callResponse) {
```

Comment:
I'm wondering if this can use the existing `isNotAuthorizedOrForbidden` method in ResponseStep and if this check should be in `onFailureNoRetry()` method instead of `onFailure`.

Reply:
I think if there is no RetryStrategy set for a API call, which is the case here, the onFailure behaves the same as the onFailureNoRetry. But I also agree that it is probably clearer to call onFailureNoRetry() directly here.

Reply:
In looking at the code in AsyncRequestStep.java, I believe if there is no RetryStrategy set,  it defaults to the same async request step.

```
      retryStrategy = Optional.ofNullable(retry)
            .orElse(new DefaultRetryStrategy(maxRetryCount, AsyncRequestStep.this, AsyncRequestStep.this));
```